### ISSUE TITLE
Don't know default credentials/accounts

### ISSUE THREAD

**Issue by privateseabass** [monicahq/monica#3762]
If your issue is about a feature request, tell us why you need it.
When I setup the server going from Debian or Ubuntu install methods, I can't tell what the initial admin's emails or passwords are supposed to be.
I'm not strong with SQL, so I don't know if I'm just not seeing it and I can't tell if I am supposed to find it somewhere.

So what are they supposed to be or how are you supposed to add it?
I would ask for it to be put in, but I'd honestly have made a pull request already if I knew these answers.

**Comment by mdeuerlein** [comment]
I had the same problem, this worked for me:

1. delete all entries from the database table "users" (eg. `DELETE FROM user`)
2. Run the command: 
`php artisan setup:production --email=your@email.com --password=yourpassword`
3. open browser and login

If this doesn't work or if you are prompted for a recovery token: 
Delete/drop all tables from your database or create a new one and run the command again.

**Comment by fklappan** [comment]
@mdeuerlein didn't work for me.
After following your steps, still the FakeUserTableSeeder is executed:

```
Command: php artisan setup:production --email=email@host.de --password=test -v

✓ Filling database
Seeding: FakeUserTableSeeder
Seeded:  FakeUserTableSeeder (19.72 seconds)
```

**Comment by asbiin** [comment]
You have to set `APP_ENV` to `production` to not have the FakeUserTableSeeder executed!

**Comment by fklappan** [comment]
Thats awkward... sorry 😢 

I'm going to improve the installation instruction, thank you!

**Comment by privateseabass** [comment]
If you have improved installation instructions to resolve this in another pull request, you can close this one.

**Comment by njm2112** [comment]
I don't think this should've been closed just yet as the problem persists.

The instructions don't seem to indicate that when adding the proper credentials for the mysql db into `.env`, the user also has to change `APP_DISABLE_SIGNUP` to `false`. Once that is done, upon loading the given monica url, you can create your first user.

**Comment by privateseabass** [comment]
I was able to create my first user without changing that option.
However, I didn't realize that was an option.
Good to know...

**Comment by asbiin** [comment]
The question is
>  I can't tell what the initial admin's emails or passwords are supposed to be

.. and there is none.

After installation, you're invited to create a user, even if `APP_DISABLE_SIGNUP` is set to `false`.

I think the doc is better now, and we may always improve it.
